svm/tests/concurrent_tests.rs

```diff
@@ -33,32 +33,57 @@ fn fast_concur_test() {
         .map(|(idx, key)| (*key, idx as u64))
         .collect();
 
-    for _ in 0..10 {
-        let ths: Vec<_> = (0..4)
-            .map(|_| {
-                let local_bank = mock_bank.clone();
-                let processor = TransactionBatchProcessor::new_from(
-                    &batch_processor,
-                    batch_processor.slot,
-                    batch_processor.epoch,
-                );
-                let maps = account_maps.clone();
-                let programs = programs.clone();
-                thread::spawn(move || {
-                    let result = processor.replenish_program_cache(&local_bank, &maps, false, true);
-                    for key in &programs {
-                        let cache_entry = result.find(key);
-                        assert!(matches!(
-                            cache_entry.unwrap().program,
-                            ProgramCacheEntryType::Loaded(_)
-                        ));
-                    }
-                })
+    let ths: Vec<_> = (0..4)
+        .map(|_| {
+            let local_bank = mock_bank.clone();
+            let processor = TransactionBatchProcessor::new_from(
+                &batch_processor,
+                batch_processor.slot,
+                batch_processor.epoch,
+            );
+            let maps = account_maps.clone();
+            let programs = programs.clone();
+            thread::spawn(move || {
+                let result = processor.replenish_program_cache(&local_bank, &maps, false, true);
+                for key in &programs {
+                    let cache_entry = result.find(key);
+                    assert!(matches!(
+                        cache_entry.unwrap().program,
+                        ProgramCacheEntryType::Loaded(_)
+                    ));
+                }
             })
-            .collect();
+        })
+        .collect();
 
-        for th in ths {
-            th.join().unwrap();
-        }
+    for th in ths {
+        th.join().unwrap();
     }
 }
+
+#[test]
+fn probabilistic_concurrency_test() {
```

Comment:
Names of `probabilistic_concurrency_test` and `random_concurrency_test` are not very telling. Also some comments explaining what they do might help.

Reply:
Fixed